question/bank/exportquestions/classes/exportquestions_helper.php

```diff
@@ -0,0 +1,58 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Library functions used by question/preview.php.
+ *
+ * @package    qbank_exportquestions
+ * @copyright  2010 The Open University
+ * @author     2021 Safat Shahin <safatshahin@catalyst-au.net>
```

Comment:
please change to your name and if the code is from questionlib, copyright should follow that.

Reply:
maybe something like this

/**
 * Helper class for adding/editing a question.
 * 
 * This code is based on question/editlib.php by Martin Dougiamas.
 *
 * @package    qbank_editquestion
 * @copyright  2021 Safat Shahin <safatshahin@catalyst-au.net>   -- your name
 * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
 */

Reply:
You're right I missed that, thank you.